queries/attachments/links_free_file_host.yml

```diff
@@ -0,0 +1,12 @@
+name: "Attachment with links to free file sharing service"
+type: "query"
+source: |
+  filter(
+    map(attachments,
+        map(beta.binexplode(.),
+            map(filter(.scan.url.urls, .domain.root_domain in $free_file_hosts), .url)
```

Comment:
it looks like we're extracting `.url` here, should this be pulling out `.file_name` instead?